README.md

```diff
@@ -9,7 +9,9 @@ Compare two objects using accessed properties with Proxy
 
 ## Introduction
 
-This is an internal library used in [React Tracked](https://react-tracked.js.org).
+This is an internal library used in: 
+1.[React Tracked](https://react-tracked.js.org).
+2.[Valtio](https://github.com/pmndrs/valtio).
```

Comment:
We already have a list down there, and `proxy-memoize` is also a featured one. How about changing it like this?
```suggestion
This is an internal library used in state management libraries such as [React Tracked](https://react-tracked.js.org) and [Valtio](https://github.com/pmndrs/valtio).
```

Reply:
It sounds better😊